internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -118,10 +123,17 @@ func (v *vmConsoleProxy) Cleanup(request *common.Request) ([]common.CleanupResul
 
 func reconcileServiceAccountsFuncs(serviceAccount core.ServiceAccount) common.ReconcileFunc {
 	return func(request *common.Request) (common.ReconcileResult, error) {
-		serviceAccount.Namespace = kubeVirtNamespace
+		serviceAccount.Namespace = getVmConsoleProxyNamespace(request)
 		return common.CreateOrUpdate(request).
 			ClusterResource(&serviceAccount).
 			WithAppLabels(operandName, operandComponent).
+			UpdateFunc(func(newRes, foundRes client.Object) {
```

Comment:
I was mistaken about this change. The cluster adds a secret to the service account by default. This default field causes a reconciliation loop, and is removed again here in this function.

The loop means that SSP is always in a `Deploying` state. Curiously, the functional tests should fail after 10 minutes if `Deployed` state is not reached, but because of a race, sometimes the `Deployed` state is seen and that is enough for the test to progress.

Please remove this `UpdateFunc()`.

Reply:
Done.